### DISCUSSION TITLE
How can I make the navbar the same width as the body of the page in a quarto website?

### DISCUSSION THREAD

**Discussion by sckott** [quarto-dev/quarto-cli#7506] [Q&A] [answered]
### Description

For context, my use case is trying to duplicate a theme from a Jekyll website as close as possible. One element of that is making the navbar and body of the page the same width. The Quarto docs don't seem to have anything along those lines, but perhaps there is a way to do it with the theme options or Sass variables?

This was the closest question I saw related to my question https://github.com/quarto-dev/quarto-cli/discussions/5489 - but I'm curious if there's a way to do this with just theme options or Sass variables rather than a fully custom css.

**Comment by mcanouil** [comment, marked as answer]
I don't think you can as the navbar is part of the envelop so not much customisation from Quarto directly, but note that sass variables, etc. are "all" described in the documentation:
- https://quarto.org/docs/output-formats/html-themes.html#layout
- https://quarto.org/docs/output-formats/page-layout.html

Beyond those, it's Bootstrap and (S)CSS.
You can use your favourite browser inspect/developer mode to target to the proper elements.

**Reply by sckott** [reply]
Thanks very much.
